File: mobile_app/app/Objects/src/com/menube/app/designerscripts/LS_frmdirecciones.java
package com.menube.app.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmdirecciones{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("paneldireccion").vw.setLeft((int)(0d));
views.get("paneldireccion").vw.setWidth((int)((100d / 100 * width) - (0d)));
views.get("paneldireccion").vw.setTop((int)(0d));
views.get("paneldireccion").vw.setHeight((int)((100d / 100 * height) - (0d)));
views.get("panel1").vw.setLeft((int)(0d));
views.get("panel1").vw.setWidth((int)((100d / 100 * width) - (0d)));
views.get("panel1").vw.setTop((int)(0d));
views.get("panel1").vw.setHeight((int)((9d / 100 * height) - (0d)));
views.get("btndireccionback").vw.setLeft((int)(0d));
views.get("btndireccionback").vw.setTop((int)(0d));
views.get("btndireccionback").vw.setWidth((int)((15d / 100 * width)));
views.get("btndireccionback").vw.setHeight((int)((views.get("panel1").vw.getHeight())));
views.get("label1").vw.setTop((int)(0d));
views.get("label1").vw.setLeft((int)((views.get("btndireccionback").vw.getLeft() + views.get("btndireccionback").vw.getWidth())));
views.get("label1").vw.setHeight((int)((views.get("btndireccionback").vw.getHeight())));
views.get("label1").vw.setWidth((int)((100d / 100 * width)-(views.get("label1").vw.getLeft())*2d));
views.get("label2").vw.setTop((int)((views.get("panel1").vw.getTop() + views.get("panel1").vw.getHeight())+(2d / 100 * height)));
views.get("label2").vw.setHeight((int)((5d / 100 * height)));
views.get("label2").vw.setLeft((int)((5d / 100 * width)));
views.get("label2").vw.setWidth((int)((views.get("paneldireccion").vw.getWidth())-(views.get("label2").vw.getLeft())));
views.get("panel2").vw.setTop((int)((views.get("label2").vw.getTop() + views.get("label2").vw.getHeight())));
views.get("panel2").vw.setLeft((int)((views.get("label2").vw.getLeft())));
views.get("panel2").vw.setHeight((int)((41.5d / 100 * height)));
views.get("panel2").vw.setWidth((int)((views.get("paneldireccion").vw.getWidth())-(views.get("panel2").vw.getLeft())*2d));
views.get("listviewdirecciones").vw.setLeft((int)(0d));
views.get("listviewdirecciones").vw.setTop((int)(0d));
views.get("listviewdirecciones").vw.setWidth((int)((views.get("panel2").vw.getWidth())));
views.get("listviewdirecciones").vw.setHeight((int)((views.get("panel2").vw.getHeight())));
views.get("label3").vw.setTop((int)((views.get("panel2").vw.getTop() + views.get("panel2").vw.getHeight())+(2d / 100 * height)));
views.get("label3").vw.setWidth((int)((views.get("label2").vw.getWidth())));
views.get("label3").vw.setHeight((int)((views.get("label2").vw.getHeight())));
views.get("label3").vw.setLeft((int)((views.get("label2").vw.getLeft())));
views.get("panel3").vw.setTop((int)((views.get("label3").vw.getTop() + views.get("label3").vw.getHeight())));
views.get("panel3").vw.setLeft((int)((views.get("panel2").vw.getLeft())));
//BA.debugLineNum = 42;BA.debugLine="Panel3.Width = Panel2.Width"[frmDirecciones/General script]
views.get("panel3").vw.setWidth((int)((views.get("panel2").vw.getWidth())));
//BA.debugLineNum = 43;BA.debugLine="Panel3.Height = 20%y"[frmDirecciones/General script]
views.get("panel3").vw.setHeight((int)((20d / 100 * height)));
//BA.debugLineNum = 45;BA.debugLine="txtDireccion.Left = 0"[frmDirecciones/General script]
views.get("txtdireccion").vw.setLeft((int)(0d));
//BA.debugLineNum = 46;BA.debugLine="txtDireccion.Top = 0"[frmDirecciones/General script]
views.get("txtdireccion").vw.setTop((int)(0d));
//BA.debugLineNum = 47;BA.debugLine="txtDireccion.Width = Panel3.Width"[frmDirecciones/General script]
views.get("txtdireccion").vw.setWidth((int)((views.get("panel3").vw.getWidth())));
//BA.debugLineNum = 48;BA.debugLine="txtDireccion.Height = Panel3.Height + 1%y"[frmDirecciones/General script]
views.get("txtdireccion").vw.setHeight((int)((views.get("panel3").vw.getHeight())+(1d / 100 * height)));
//BA.debugLineNum = 50;BA.debugLine="chPrincipal.Width = Panel3.Width"[frmDirecciones/General script]
views.get("chprincipal").vw.setWidth((int)((views.get("panel3").vw.getWidth())));
//BA.debugLineNum = 51;BA.debugLine="chPrincipal.Height = 5%y"[frmDirecciones/General script]
views.get("chprincipal").vw.setHeight((int)((5d / 100 * height)));
//BA.debugLineNum = 52;BA.debugLine="chPrincipal.Top = Panel3.Bottom"[frmDirecciones/General script]
views.get("chprincipal").vw.setTop((int)((views.get("panel3").vw.getTop() + views.get("panel3").vw.getHeight())));
//BA.debugLineNum = 53;BA.debugLine="chPrincipal.Left = Panel3.Left"[frmDirecciones/General script]
views.get("chprincipal").vw.setLeft((int)((views.get("panel3").vw.getLeft())));
//BA.debugLineNum = 55;BA.debugLine="btnNuevaDireccion.Left = Panel3.Left"[frmDirecciones/General script]
views.get("btnnuevadireccion").vw.setLeft((int)((views.get("panel3").vw.getLeft())));
//BA.debugLineNum = 56;BA.debugLine="btnNuevaDireccion.Height = 7%y"[frmDirecciones/General script]
views.get("btnnuevadireccion").vw.setHeight((int)((7d / 100 * height)));
//BA.debugLineNum = 57;BA.debugLine="btnNuevaDireccion.Width = 44%x"[frmDirecciones/General script]
views.get("btnnuevadireccion").vw.setWidth((int)((44d / 100 * width)));
//BA.debugLineNum = 58;BA.debugLine="btnNuevaDireccion.Top = chPrincipal.Bottom + 1%y"[frmDirecciones/General script]
views.get("btnnuevadireccion").vw.setTop((int)((views.get("chprincipal").vw.getTop() + views.get("chprincipal").vw.getHeight())+(1d / 100 * height)));
//BA.debugLineNum = 60;BA.debugLine="btnSalvarDireccion.Width = btnNuevaDireccion.Width"[frmDirecciones/General script]
views.get("btnsalvardireccion").vw.setWidth((int)((views.get("btnnuevadireccion").vw.getWidth())));
//BA.debugLineNum = 61;BA.debugLine="btnSalvarDireccion.Height = btnNuevaDireccion.Height"[frmDirecciones/General script]
views.get("btnsalvardireccion").vw.setHeight((int)((views.get("btnnuevadireccion").vw.getHeight())));
//BA.debugLineNum = 62;BA.debugLine="btnSalvarDireccion.Top = btnNuevaDireccion.Top"[frmDirecciones/General script]
views.get("btnsalvardireccion").vw.setTop((int)((views.get("btnnuevadireccion").vw.getTop())));
//BA.debugLineNum = 63;BA.debugLine="btnSalvarDireccion.Left = btnNuevaDireccion.Right +2%x"[frmDirecciones/General script]
views.get("btnsalvardireccion").vw.setLeft((int)((views.get("btnnuevadireccion").vw.getLeft() + views.get("btnnuevadireccion").vw.getWidth())+(2d / 100 * width)));

}
}

File: mobile_app/app_ios/Objects/src/B4iProject/LS_frmdirecciones.m

#import "iCore.h"
@interface LS_frmdirecciones:NSObject
@end

@implementation LS_frmdirecciones

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
//BA.debugLineNum = 2;BA.debugLine="AutoScaleAll"[frmDirecciones/General script]
[B4ILayoutBuilder scaleAll:views :width :height];
//BA.debugLineNum = 4;BA.debugLine="Label2.Top =  2%y"[frmDirecciones/General script]
[views get:@"label2"].Top = (int)((2 / 100.0 * height));
//BA.debugLineNum = 5;BA.debugLine="Label2.Height = 5%y"[frmDirecciones/General script]
[views get:@"label2"].Height = (int)((5 / 100.0 * height));
//BA.debugLineNum = 6;BA.debugLine="Label2.Left = 5%x"[frmDirecciones/General script]
[views get:@"label2"].Left = (int)((5 / 100.0 * width));
//BA.debugLineNum = 7;BA.debugLine="Label2.Width = 100%x - Label2.Left"[frmDirecciones/General script]
[views get:@"label2"].Width = (int)((100 / 100.0 * width)-([views get:@"label2"].Left));
//BA.debugLineNum = 9;BA.debugLine="Panel2.Top = Label2.Bottom"[frmDirecciones/General script]
[views get:@"panel2"].Top = (int)(([views get:@"label2"].Top + [views get:@"label2"].Height));
//BA.debugLineNum = 10;BA.debugLine="Panel2.Left = Label2.Left"[frmDirecciones/General script]
[views get:@"panel2"].Left = (int)(([views get:@"label2"].Left));
//BA.debugLineNum = 11;BA.debugLine="Panel2.Height = 48.5%y"[frmDirecciones/General script]
[views get:@"panel2"].Height = (int)((48.5 / 100.0 * height));
//BA.debugLineNum = 12;BA.debugLine="Panel2.Width = 100%x - Panel2.Left * 2"[frmDirecciones/General script]
[views get:@"panel2"].Width = (int)((100 / 100.0 * width)-([views get:@"panel2"].Left)*2.0);
//BA.debugLineNum = 14;BA.debugLine="Label3.Top = Panel2.Bottom +2%y"[frmDirecciones/General script]
[views get:@"label3"].Top = (int)(([views get:@"panel2"].Top + [views get:@"panel2"].Height)+(2 / 100.0 * height));
//BA.debugLineNum = 15;BA.debugLine="Label3.Width = Label2.Width"[frmDirecciones/General script]
[views get:@"label3"].Width = (int)(([views get:@"label2"].Width));
//BA.debugLineNum = 16;BA.debugLine="Label3.Height = Label2.Height"[frmDirecciones/General script]
[views get:@"label3"].Height = (int)(([views get:@"label2"].Height));
//BA.debugLineNum = 17;BA.debugLine="Label3.Left = Label2.Left"[frmDirecciones/General script]
[views get:@"label3"].Left = (int)(([views get:@"label2"].Left));
//BA.debugLineNum = 19;BA.debugLine="txtDireccion.Top = Label3.Bottom"[frmDirecciones/General script]
[views get:@"txtdireccion"].Top = (int)(([views get:@"label3"].Top + [views get:@"label3"].Height));
//BA.debugLineNum = 20;BA.debugLine="txtDireccion.Left = Panel2.Left"[frmDirecciones/General script]
[views get:@"txtdireccion"].Left = (int)(([views get:@"panel2"].Left));
//BA.debugLineNum = 21;BA.debugLine="txtDireccion.Width = Panel2.Width"[frmDirecciones/General script]
[views get:@"txtdireccion"].Width = (int)(([views get:@"panel2"].Width));
//BA.debugLineNum = 22;BA.debugLine="txtDireccion.Height = 20%y"[frmDirecciones/General script]
[views get:@"txtdireccion"].Height = (int)((20 / 100.0 * height));
//BA.debugLineNum = 24;BA.debugLine="chPrincipal.Width = 60dip"[frmDirecciones/General script]
[views get:@"chprincipal"].Width = (int)((60 * 1));
//BA.debugLineNum = 25;BA.debugLine="chPrincipal.Height = 5%y"[frmDirecciones/General script]
[views get:@"chprincipal"].Height = (int)((5 / 100.0 * height));
//BA.debugLineNum = 26;BA.debugLine="chPrincipal.Top = txtDireccion.Bottom + 1%y"[frmDirecciones/General script]
[views get:@"chprincipal"].Top = (int)(([views get:@"txtdireccion"].Top + [views get:@"txtdireccion"].Height)+(1 / 100.0 * height));
//BA.debugLineNum = 27;BA.debugLine="chPrincipal.Left = txtDireccion.Left"[frmDirecciones/General script]
[views get:@"chprincipal"].Left = (int)(([views get:@"txtdireccion"].Left));
//BA.debugLineNum = 29;BA.debugLine="Label1.Left = chPrincipal.Right"[frmDirecciones/General script]
[views get:@"label1"].Left = (int)(([views get:@"chprincipal"].Left + [views get:@"chprincipal"].Width));
//BA.debugLineNum = 30;BA.debugLine="Label1.Width = 100%x - Label1.Left"[frmDirecciones/General script]
[views get:@"label1"].Width = (int)((100 / 100.0 * width)-([views get:@"label1"].Left));
//BA.debugLineNum = 31;BA.debugLine="Label1.Height = chPrincipal.Height"[frmDirecciones/General script]
[views get:@"label1"].Height = (int)(([views get:@"chprincipal"].Height));
//BA.debugLineNum = 32;BA.debugLine="Label1.Top = chPrincipal.Top"[frmDirecciones/General script]
[views get:@"label1"].Top = (int)(([views get:@"chprincipal"].Top));
//BA.debugLineNum = 34;BA.debugLine="btnNuevaDireccion.Left = txtDireccion.Left"[frmDirecciones/General script]
[views get:@"btnnuevadireccion"].Left = (int)(([views get:@"txtdireccion"].Left));
//BA.debugLineNum = 35;BA.debugLine="btnNuevaDireccion.Height = 7%y"[frmDirecciones/General script]
[views get:@"btnnuevadireccion"].Height = (int)((7 / 100.0 * height));
//BA.debugLineNum = 36;BA.debugLine="btnNuevaDireccion.Width = 44%x"[frmDirecciones/General script]
[views get:@"btnnuevadireccion"].Width = (int)((44 / 100.0 * width));
//BA.debugLineNum = 37;BA.debugLine="btnNuevaDireccion.Top = chPrincipal.Bottom + 2%y"[frmDirecciones/General script]
[views get:@"btnnuevadireccion"].Top = (int)(([views get:@"chprincipal"].Top + [views get:@"chprincipal"].Height)+(2 / 100.0 * height));
//BA.debugLineNum = 39;BA.debugLine="btnSalvarDireccion.Width = btnNuevaDireccion.Width"[frmDirecciones/General script]
[views get:@"btnsalvardireccion"].Width = (int)(([views get:@"btnnuevadireccion"].Width));
//BA.debugLineNum = 40;BA.debugLine="btnSalvarDireccion.Height = btnNuevaDireccion.Height"[frmDirecciones/General script]
[views get:@"btnsalvardireccion"].Height = (int)(([views get:@"btnnuevadireccion"].Height));
//BA.debugLineNum = 41;BA.debugLine="btnSalvarDireccion.Top = btnNuevaDireccion.Top"[frmDirecciones/General script]
[views get:@"btnsalvardireccion"].Top = (int)(([views get:@"btnnuevadireccion"].Top));
//BA.debugLineNum = 42;BA.debugLine="btnSalvarDireccion.Left = btnNuevaDireccion.Right +2%x"[frmDirecciones/General script]
[views get:@"btnsalvardireccion"].Left = (int)(([views get:@"btnnuevadireccion"].Left + [views get:@"btnnuevadireccion"].Width)+(2 / 100.0 * width));

}
@end